tutorial.proto

```diff
@@ -32,37 +58,45 @@ message InlineContent {
 };
 
 message Heading {
-  repeated InlineContent inline_contents = 1;
+  string text = 3;
   int32 level = 2;
+
+  reserved 1;
 };
 
 message Image {
-  // TODO
+  string source = 1;
+  string size = 2;
+  string alt_title = 3;
+  string alt_description = 3;
 };
 
+message YoutubeVideo {
+  Image ImageSource = 1;
+}
+
 message InfoBox {
   repeated InlineContent inline_contents = 1;
 
   enum Disposition {
     UNKNOWN_DISPOSITION = 0;
-    POSITIVE = 1;
-    NEGATIVE = 2;
+    // POSITIVE = 1;
+    // NEGATIVE = 2;
```

Comment:
Almost all ui/md framework use 'note, warning, failure', etc as descriptors. Can send internal link to g3doc of similar ui elements if desired.